### DISCUSSION TITLE
Grafana - Capture Loss data

### DISCUSSION THREAD

**Discussion by presianbg** [Security-Onion-Solutions/securityonion#6989] [Unsupported Versions] [answered]
Hi,

Please help me to understand what problem is lurking behind `Capture Loss` graph:
![image](https://user-images.githubusercontent.com/2163486/151134971-9ef4c88a-0063-48c0-9066-47115b128003.png)

and how to troubleshoot / fix it ?

In the very same time I have no Zeek / Suricata loss:
![image](https://user-images.githubusercontent.com/2163486/151135360-ff0c4492-341e-487a-8ff5-b829d6c2e5fc.png)

Steno is disabled.

Cheers,
PY

**Comment by dougburks** [comment, marked as answer]
If Zeek is reporting capture loss but no packet loss, this usually means that the capture loss is happening upstream in the tap or span port itself. I've added this to https://docs.securityonion.net/en/2.3/zeek.html#packet-loss-and-capture-loss.
